src/main/java/alpha/DeleteCommand.java

```diff
@@ -0,0 +1,18 @@
+package alpha;
+
+public class DeleteCommand extends Command {
+
+    private int index;
+
+    public DeleteCommand(TaskList taskList, FileHandler fileHandler, UI ui, int index) {
+        super(taskList, fileHandler, ui);
+        isExit = false;
+        this.index = index;
+    }
```

Comment:
Create JavaDoc comments for this and other classes as well, as you have done in Alpha.java and Command.java